.devcontainer/Dockerfile.console

```
@@ -0,0 +1,19 @@
+FROM openshift/origin-cli:latest AS cli
+FROM quay.io/coreos/tectonic-console-builder:v23 AS build
+SHELL ["/bin/bash", "-c"]
+
+COPY --from=cli /usr/bin/oc /usr/local/bin/oc
+RUN git clone https://github.com/openshift/console /console
+
+WORKDIR /console
+RUN [ ! -f "./bin/bridge" ] && ./build-backend.sh
+RUN [ ! -d "./frontend/public/dist" ] && ./build-frontend.sh
```

Always build. There is no way to know if the existing binaries are built for the correct architecture.

```suggestion
RUN ./build.sh
```